src/socialMediaShare/SharedButtons.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {
+	FacebookIcon,
+	LineIcon,
+	LinkedinIcon,
+	TwitterIcon,
+	WhatsappIcon,
+	WeiboIcon,
+	RedditIcon,
+	InstapaperIcon,
+	FacebookShareButton,
+	LineShareButton,
+	LinkedinShareButton,
+	TwitterShareButton,
+	WhatsappShareButton,
+	WeiboShareButton,
+	RedditShareButton,
+	InstapaperShareButton,
+} from 'react-share';
+
+import { SharedContent } from './constants.js';
+
+const SharedButtons = () => {
+	return (
+		<div>
```

Comment:
We have different shared contents for different platforms, already DMed you, could please set them accordingly?